engine/src/test/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessorTest.java

```diff
@@ -299,46 +351,185 @@ void oBEndpointProcessingGoodBridge() {
+
+        // Assert the results for the fields in the "data" object.
+        final JsonObject data = json.getJsonObject("data");
+        Assertions.assertTrue(data.containsKey("context"), "the \"context\" field is not present");
+        Assertions.assertTrue(data.containsKey("events"), "the \"events\" field is not present");
+
+        // Assert the values for the fields in the "data" object.
+        Assertions.assertEquals(FIXTURE_ACTION_ACCOUNT_ID, data.getString("account_id"), "the \"account_id\" values don't match");
+        Assertions.assertEquals(FIXTURE_ACTION_APP, data.getString("application"), "the \"application\" values don't match");
+        Assertions.assertEquals(FIXTURE_ACTION_BUNDLE, data.getString("bundle"), "the \"bundle\" values don't match");
+        Assertions.assertEquals(FIXTURE_ACTION_EVENT_TYPE, data.getString("event_type"), "the \"event_type\" values don't match");
+        Assertions.assertEquals(FIXTURE_ACTION_ORG_ID, data.getString("org_id"), "the \"org_id\" values don't match");
+        Assertions.assertEquals(FIXTURE_ACTION_TIMESTAMP.toString(), data.getString("timestamp"), "the \"timestamp\" values don't match");
+
+        // Assert the fields and the values in the "context" object.
+        final JsonObject context = data.getJsonObject("context");
+        Assertions.assertTrue(context.containsKey(FIXTURE_ACTION_CONTEXT), String.format("the expected \"%s\" context field is not present", FIXTURE_ACTION_CONTEXT));
+        Assertions.assertEquals(FIXTURE_ACTION_CONTEXT_VALUE, context.getString(FIXTURE_ACTION_CONTEXT), String.format("the \"%s\" context values don't match", FIXTURE_ACTION_CONTEXT_VALUE));
+
+        // Assert the "events" array.
+        final JsonArray events = data.getJsonArray("events");
+        if (events.size() != 1) {
+            Assertions.fail(String.format("the events array has an unexpected number of elements. Want %s, got %s", 1, events.size()));
+        }
+
+        final JsonObject eventResult = events.getJsonObject(0);
+
+        Assertions.assertTrue(eventResult.containsKey("metadata"), "the \"metadata\" field is not present");
+        Assertions.assertTrue(eventResult.containsKey("payload"), "the \"payload\" field is not present");
+
+        // Assert the "payload" object in the event.
+        final JsonObject payload = eventResult.getJsonObject("payload");
+        Assertions.assertTrue(payload.containsKey(FIXTURE_PAYLOAD_ADDITIONAL_PROPERTY), String.format("the \"%s\" payload field is not present", FIXTURE_PAYLOAD_ADDITIONAL_PROPERTY));
+        Assertions.assertTrue(payload.containsKey(FIXTURE_PAYLOAD_ADDITIONAL_PROPERTY_2), String.format("the \"%s\" payload field is not present", FIXTURE_PAYLOAD_ADDITIONAL_PROPERTY_2));
+        Assertions.assertTrue(payload.containsKey(FIXTURE_PAYLOAD_ADDITIONAL_PROPERTY_3), String.format("the \"%s\" payload field is not present", FIXTURE_PAYLOAD_ADDITIONAL_PROPERTY_3));
```

Comment:
These could be removed since there are value checks below.
```suggestion
```

Reply:
Thanks! Done!